packages/contracts/docs/osx/02-how-to-guides/02-plugin-development/index.md

```diff
@@ -23,32 +84,39 @@ contract GreeterPlugin is Plugin {
   constructor(IDAO _dao) Plugin(_dao) {}
 
   function greet() external pure returns (string memory) {
-    return 'Hello, world!';
+    return 'Hello world!';
```

Comment:
Shouldn't this plugin look more like the storage plugin above?

Reply:
already updated!